server/models/results.js

```diff
@@ -4,7 +4,8 @@ export default (sequelize, DataTypes) => {
     result_id: {
       allowNull: false,
       primaryKey: true,
-      type: DataTypes.INTEGER
+      type: DataTypes.INTEGER,
+      defaultValue: 0,
```

Comment:
Can we set the defaultValue to 1? I think it sounds better than having it to be zero.